styles.css

```diff
@@ -0,0 +1,16 @@
+@media (min-width: 768px) {
```

Comment:
Do we need any styles at all? style like this tend to be very brittle when mixed with the variety of client themes

Reply:
without the css:
![image](https://user-images.githubusercontent.com/57339730/70590019-17f84f80-1c1d-11ea-9d7d-0f1c771e7a6e.png)

with the css:
![image](https://user-images.githubusercontent.com/57339730/70590078-4d9d3880-1c1d-11ea-972c-765652748eac.png)

Reply:
css removed